zappa/__init__.py

```diff
@@ -12,7 +12,7 @@ def running_in_docker() -> bool:
     return running_in_docker_flag
 
 
-SUPPORTED_VERSIONS = [(3, 7), (3, 8), (3, 9)]
+SUPPORTED_VERSIONS = [(3, 7), (3, 8), (3, 9),(3,10)]
```

Comment:
I don't know zappa well so I'm not sure if 3.10 is a supported version,
but I think it would be nice to have a blank space.

before
`SUPPORTED_VERSIONS = [(3, 7), (3, 8), (3, 9),(3,10)]`

after
`SUPPORTED_VERSIONS = [(3, 7), (3, 8), (3, 9), (3, 10)]`